content/docs/guides/observability/metrics.md

```diff
@@ -249,8 +249,9 @@ Ensure that you have followed the steps to run [OSM Demo][2]
 ### Querying proxy metrics for request count
 
 1. Verify that the Prometheus service is running in your cluster
-   - In kubernetes, execute the following command: `kubectl get svc osm-prometheus -n osm-system`
+   - In kubernetes, execute the following command: `kubectl get svc osm-prometheus -n "<osm-namespace>"`.
```

Comment:
Not sure if the quotations around `<osm-namespace>` are intentional.
```suggestion
   - In kubernetes, execute the following command: `kubectl get svc osm-prometheus -n <osm-namespace>`.
```

Reply:
They aren't, thanks!